BoogieLang/Ast_Cfg_Transformation.thy

```diff
@@ -0,0 +1,2280 @@
+section \<open>Generic lemmas used to validate AST-to-CFG phase\<close>
+
+theory Ast_Cfg_Transformation
+   imports Main
+          "Boogie_Lang.Ast"
+          "Boogie_Lang.Semantics"
+          "Boogie_Lang.BackedgeElim"
```

Comment:
Remove Boogie_Lang reference, since files are referring to the same folder.
"Boogie_Lang.Ast" -> Ast